pkg/v1/tkg/kind/client.go

```diff
@@ -394,3 +376,38 @@ func (k *KindClusterProxy) getIPFamily() (kindv1.ClusterIPFamily, error) {
 		return "", fmt.Errorf("TKG_IP_FAMILY should be one of %s, %s, %s, got %s", kindv1.IPv4Family, kindv1.IPv6Family, kindv1.DualStackFamily, normalisedIPFamily)
 	}
 }
+
+func (k *KindClusterProxy) setupProxyConfigurationForKindCluster() {
+	var httpProxy, httpsProxy, noProxy, tkgHttpProxy, tkgHttpsProxy, tkgNoProxy string
+
+	httpProxy, _ = k.options.Readerwriter.Get(constants.HTTPProxy)
```

Comment:
nit: this is also a behavioral change, I suggest a small comment to say we are respecting the non-"TKG_" proxy settings as well.

Reply:
Sure. Added description as part of the function.